README.md

```diff
@@ -276,3 +276,6 @@ On login, Dink can submit a character summary containing data that spans multipl
 ## Credits
 
 This plugin uses code from [Universal Discord Notifier](https://github.com/MidgetJake/UniversalDiscordNotifier).
+
+Item rarity data is sourced from the OSRS Wiki (licensed under [CC BY-NC-SA 3.0](https://creativecommons.org/licenses/by-nc-sa/3.0/)),
+which was conveniently parsed by [Flipping Utilities](https://github.com/Flipping-Utilities/parsed-osrs) (and transformed by pajlads).
```

Comment:
Could we include the transformation util as a utility in this repo, or make a new repo for it & link to it?
I can create it under pajlads if you don't have access

Reply:
Done in https://github.com/pajlads/DinkPlugin/pull/425/commits/b07ea224c585ce15482b4685cfa9ac1c4a9ff1a9

We can now run `./gradlew generateResources` to update `npc_drops.json` - a separate PR should be created to regularly execute this task from a gh action